test/packages/good/deps.yml

```diff
@@ -0,0 +1,4 @@
+build:
```

Comment:
I'm ok with `build.yml` too, although this may suggest to introduce more side features into it :) We have also `_dev` directory, maybe this one is a better fit? For example: `_dev/build/deps.yml`. WDYT?

Reply:
I adjusted the PR to use `build.yml` in the package root directory